app/(main)/(sources)/sources.tsx

```
@@ -62,14 +59,13 @@ const Sources = () => {
       setLoading(false);
     } catch (e) {
-      // addToast("Something went wrong while fetching pages", { appearance: "error" })
       Toaster.error("Something went wrong while fetching pages");
     }
   };
 
   useEffect(() => {
-    getPages("TEMP");
-  }, []);
+    getPages();
+  }, [, isFocused]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)